### ISSUE TITLE
Alternative Color Scheme For ColourBlindness

### ISSUE THREAD

**Issue by hhamud** [cryptonative-ch/mesa-interface#253]
## Detailed Description
Alternative Color Scheme For ColourBlindness

## Context
Great for Accessibility

**Comment by martinkrung** [comment]
I agree on this, I have some limited Colour vision. But we should not have alternative color scheme, we should fix the existing one. Because from enhanced accessibility all people profit.
